doc/user/content/ops/network-security/privatelink.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "AWS MSK - PrivateLink"
+description: "How to connect Materialize to an AWS MSK cluster using a Privatelink connection"
+menu:
+  main:
+    parent: "network-security"
+    name: "AWS MSK - PrivateLink"
+---
+
+Materialize can connect to an AWS MSK cluster over a PrivateLink connection.
+
+In this guide, we'll cover how to:
+
+* Configure an AWS PrivateLink connection
+* Create `AWS PRIVATELINK` and `KAFKA` connections in Materialize
+* Create an MSK Kafka source that uses the PrivateLink connection
+
+### Prerequisites
+
+Before moving on, double-check that you have:
+
+* A running [AWS MSK Cluster](/integrations/aws-msk)
+* A region enabled Materialize
+
+### Steps
+
+1. #### Create Target Groups
+    Create [target groups](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/create-target-group.html) dedicated for each broker (in this example we will use: `b-1`, `b-2`) with the following details:
+
+    a. Target type as **IP address**.
+
+    b. Protocol as **TCP**.
+
+    c. Port as **9092,** or the port that you are using in case it is not 9092, e.g. 9094 for TLS or 9096 for SASL, etc.
+
+    d. Make sure that the target group is in the same VPC as the MSK cluster.
+
+    e. Click next, and register the respective MSK broker to each target group by its IP address.
+
+    > Make sure to create a separate target group for each broker.
+
+1. #### Security Groups and Health Checks
+
+    Once the target groups have been created, make sure that the [health checks](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/target-group-health-checks.html) are passing and that the targets are reported as healthy.
+
+    If you have set up a security group for your MSK cluster, you must ensure that the security groups allow traffic on both the listener port and the health check port.
+
+    **Remarks**:
+
+    a. Network Load Balancers do not have associated security groups. Therefore, the security groups for your targets must use IP addresses to allow traffic.
```

Comment:
Should this remark be here or after point 3.? (Were you talking about NLBs)

Reply:
Good point, but you need that information when you update your existing security groups (if any) right after setting up the target groups. The information itself is not very useful when creating the NLB itself.